cli/src/test.rs

```diff
@@ -916,4 +917,10 @@ code with ----
             }
         );
     }
+
+    #[ignore]
+    #[test]
+    fn test_print_diff_key() -> () {
+        print_diff_key()
+    }
```

Comment:
I don't think this is needed, we can just remove this - any future changes would be sparse and trivial to do something like this locally